training/splitting_dataset.py

```diff
@@ -27,7 +27,8 @@ def _get_book_names_from_dataframe(dataframe: DataFrameType) -> List[str]:
 
 def _get_grouped_book_names(dataframe: pd.DataFrame) -> Dict[int, List[str]]:
```

Comment:
Jak podbijecie do Pythona 3.10, to będzie można zamienić te `Dict` na `dict` itp.

Reply:
Czy trzeba będzie to też zmienić w innych plikach pythonowych, żeby zachować spójność stylu?